### DISCUSSION TITLE
github account indicated 404 on browser.

### DISCUSSION THREAD

**Discussion by spookeyy** [community/community#120695] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I have been trying to access my github account from several devices and different google accounts but I haven't been able to access it. I can only access from the email address associated with it and from that particular google profile alone. So when am trying to access from another device or google profile, am getting a 404 (this is not the page you are looking for).
Additionally, on site description, am seeing this; ----->  A brute force attack uses trial-and-error to guess login info, encryption keys, or find a hidden web page. Hackers work through all possible combinations hoping ...))
Clearly I don't know what happened to it or hacked maybe.

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @spookeyy ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Comment by ebndev** [comment]
Hi @spookeyy,

Thank you for being a part of the GitHub community!

Since you mentioned also being unable to access your Google account, perhaps sorting that out with them will help you with accessing your GitHub account. However, if you would like to open a ticket with GitHub's support team, the only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
